es/es-server/src/main/java/org/elasticsearch/cluster/InternalClusterInfoService.java

```diff
@@ -272,21 +272,27 @@ private void maybeRefresh() {
         }
     }
 
+    // allow tests to adjust the node stats on receipt
+    List<NodeStats> adjustNodesStats(List<NodeStats> nodeStats) {
```

Comment:
I think I don't understand what's going on here. This method is only returning the argument back without doing anything?